Compile error when referencing both xunit.core and xunit.assert.source

Since xunit 2.7.0, there is a compile error if a project uses both the xunit.core and xunit.assert.source packages:

```txt
[...]\packages\xunit.assert.source\2.7.0\contentFiles\cs\netstandard1.1\Asserts\Sdk\Exceptions\CollectionException.cs(34,41): error CS0117: 'ExceptionUtility' does not contain a definition for 'TransformStackTrace' [...]
[...]\packages\xunit.assert.source\2.7.0\contentFiles\cs\netstandard1.1\Asserts\Sdk\Exceptions\CollectionException.cs(34,78): error CS0117: 'ExceptionUtility' does not contain a definition for 'FilterStackTrace' [...]
```

This is due to the fact that an "ExceptionUtility" class exists in both the xunit.assert.source (`Xunit.Internal.ExceptionUtility`) and the xunit.execution.dotnet (`Xunit.Sdk.ExceptionUtility`) packages.

You can find a repro showing this issue here: https://github.com/ap0llo/repro-xunit-2-7-compile-error/

I've prepared a fix for this here: https://github.com/xunit/assert.xunit/pull/66/files

--- a/Sdk/Exceptions/CollectionException.cs
+++ b/Sdk/Exceptions/CollectionException.cs
@@ -10,6 +10,11 @@ using System.Globalization;
 using System.Linq;
 using Xunit.Internal;
 
+// An "ExceptionUtility" class exists in both xunit.assert (Xunit.Internal.ExceptionUtility) as well as xunit.execution.dotnet (Xunit.Sdk.ExceptionUtility)
+// This causes an compile-time error in projects that reference both the xunit.assert.source and xunit.core packages.
+// To resolve this issue, add an alias for the Xunit.Internal.ExceptionUtility to make sure, the xunit.assert core uses the right ExceptionUtility
+using ExceptionUtilityInternal = Xunit.Internal.ExceptionUtility;
+
 namespace Xunit.Sdk
 {
 	/// <summary>
@@ -31,7 +36,7 @@ namespace Xunit.Sdk
 		static string FormatInnerException(Exception innerException)
 		{
 			var text = innerException.Message;
-			var filteredStack = ExceptionUtility.TransformStackTrace(ExceptionUtility.FilterStackTrace(innerException.StackTrace), "  ");
+			var filteredStack = ExceptionUtilityInternal.TransformStackTrace(ExceptionUtilityInternal.FilterStackTrace(innerException.StackTrace), "  ");
 			if (!string.IsNullOrWhiteSpace(filteredStack))
 			{
 				if (text.Length != 0)

